linux_pipeline/Jenkinsfile_ubuntu_azure_kernel_validation

```diff
@@ -0,0 +1,226 @@
+                                            -LISAImagesShareUrl '${LISA_IMAGES_SHARE_URL}' 
+                                            -XmlTest 'ubuntu_azure_kernel_fvt.xml'
+                                            -LisaTestDependencies '${LISA_TEST_DEPENDENCIES}'
+                                            -LisaOptionalParams '${testParams}'
+                                        """)
+                                deleteDir()
+                                }  
+                            }
+                        }
+                    }
+                }
+                parallel runs
+            } else {
+                echo "Skipping Hyper-V validation"
+            }
+        }
+    },
+    "Azure": {
+        node ("azure") {
+            if (("${env:platform}" == "${PLATFORM_BOTH}") || ("${env:platform}" == "${PLATFORM_AZURE}")) {
+                def runNumber = 1
+                def runs = [:]
+                distroToTest.remove('${DISTRO_XENIAL_EDGE}')
+                distroToTest.remove('${DISTRO_BIONIC_EDGE}')
+                distroToTest.each() {
+                    def distro = it
+                    runs["${distro}-Azure"] = {
+                        stage ("${distro}-Azure") {
+                        withCredentials(bindings: [
+                        file(credentialsId:'Azure_Secrets_File', variable: 'Azure_Secrets_File')]) {
+                            echo "Azure is running"
+                            dir ("${distro}-${env.BUILD_NUMBER}-${env.BRANCH_NAME}") {
+                                git "${AZURE_AUTOMATION_GIT}"
+                                if ("${distro}" == "${DISTRO_XENIAL}") {
+                                    armVMImage = "${ARM_XENIAL}"
+                                }
+                                if ("${distro}" == "${DISTRO_BIONIC}") {
+                                    armVMImage = "${ARM_BIONIC}"
+                                }
+                                PowerShellWrapper("""
+                                    & '.\\RunAzureTests.ps1'
+                                        -BuildNumber '${BUILD_NUMBER}'
+                                        -testLocation 'westus2'
+                                        -DistroIdentifier '${distro}'
+                                        -testCycle 'PROVISION'
+                                        -testIterations '1'
+                                        -customSecretsFilePath '${Azure_Secrets_File}'
+                                        -ArchiveLogDirectory 'F:\\Logs_Azure'
+                                        -StorageAccount 'ExistingStorage_Standard'
+                                        -ARMImageName '${armVMImage}'
+                                        -OverrideVMSize 'Standard_D1_v2'
+                                        -customKernel 'proposed'
+                                        -coureCountExceededTimeout 3600
```

Comment:
typo

Reply:
It's not a typo, this is the param name in RunAzureTests.ps1